controllers/eventbus/installer/assets/jetstream/nats.conf

```diff
@@ -29,6 +29,12 @@ cluster {
   routes: [{{.Routes}}]
   cluster_advertise: $CLUSTER_ADVERTISE
   connect_retries: 120
+
+  tls {
+    cert_file: "/etc/nats-config/cluster-server-cert.pem"
+    key_file:  "/etc/nats-config/cluster-server-key.pem"
+    ca_file:   "/etc/nats-config/cluster-ca-cert.pem"
+  }
```

Comment:
In some ways this belongs in server-auth.conf. I tried adding a second "cluster" section into there but I don't think NATS can handle the two sections.